picasso/src/main/java/com/squareup/picasso3/BitmapHunter.java

```diff
@@ -206,24 +197,13 @@ Result hunt() throws IOException {
       Bitmap bitmap = result.getBitmap();
       stats.dispatchBitmapDecoded(bitmap);
 
-      int exifOrientation = result.getExifRotation();
-      if (data.needsTransformation() || exifOrientation != 0) {
-        if (data.needsMatrixTransform() || exifOrientation != 0) {
-          bitmap = transformResult(data, bitmap, exifOrientation);
-          if (picasso.loggingEnabled) {
-            log(OWNER_HUNTER, VERB_TRANSFORMED, data.logId());
-          }
-        }
-
-        result = new Result(bitmap, result.getLoadedFrom(), exifOrientation);
-        if (data.hasCustomTransformations()) {
-          result = applyCustomTransformations(data.transformations, result);
-          if (picasso.loggingEnabled) {
-            log(OWNER_HUNTER, VERB_TRANSFORMED, data.logId(),
-                "from custom transformations");
-          }
-        }
+      List<Transformation> transformations = new ArrayList<>();
```

Comment:
can be pre-sized to `data.transformations.size() + 1`. one extra slot is better than incurring expansion